docs/UserGuide.md

```diff
@@ -202,7 +223,36 @@ Example:
 
 ### Saving data
 
-Automatically saved after every change.
+Your data automatically saved after every change. The file is saved in the ubiquitous `.json` format, which
+allows you to edit the file manually without even opening **TBM** up.
+The location for this file is `./data/tbmManager.json` (indicated at the bottom left of the window), where the
+ `.` represents the directory where you have saved `TBM.jar`. Here's a snippet of this editable text file:
```

Comment:
I think the standardization for the md files is to not have line breaks unless the actual line is 2 lines, since word wrap can already do this kind of line wrapping for you